includes/connect/woocommerce-admin.php

```diff
@@ -18,8 +18,8 @@
 wc_calypso_bridge_connect_page(
 	array(
 		'screen_id' => 'woocommerce-revenue',
-		'menu'      => 'wc-admin&path=/analytics/revenue',
-		'submenu'   => 'wc-admin&path=/analytics/revenue',
+		'menu'      => 'wc-admin&path=/analytics/overview',
```

Comment:
@samueljseay I think we might need to add in an entire new call to `wc_calypso_bridge_connect_page()` with this config, and keep the legacy one around too to cover both the use cases of home screen active and not active. I just tested this branch out locally and confirmed that the analytics menu is now shown 🎉

When I opted out of the new home screen via WooCommerce > Settings > Advanced > features - The analytics menu item is no longer shown.

Reply:
@timmyc I've implemented the suggested change and tested it, it looks like it works. Would you be ok to take another look?